### PR TITLE
Update to workshop for AD changes

### PR DESCRIPTION
**Pull request by aeslaughter**
(refs #13617)

### CODE REVIEW

File: tutorials/darcy_thermo_mech/doc/content/systems/mooseobject.md [hunk 1]
```diff
@@ -60,58 +60,3 @@ CustomObject::doSomething()
   return _scale;
 }
 ```
-
```
Comment (aeslaughter) [inline tutorials/darcy_thermo_mech/doc/content/systems/mooseobject.md:63]: This deletion makes me happy

Comment (lindsayad) [reply tutorials/darcy_thermo_mech/doc/content/systems/mooseobject.md:63]: Wait...why is this deleted? This looks correct and current... AD objects return ADReals while non-AD objects return Reals

**Comment by aeslaughter** [comment]
It doesn't seem like we need to repeat the object structure because of
return type on a method. We have differing return types on many objects
already. The reason this was there originally was because of the template
structure.

**Comment by lindsayad** [comment]
> It doesn't seem like we need to repeat the object structure because of return type on a method. We have differing return types on many objects already. The reason this was there originally was because of the template structure.

I can get on board with that. Although it seems like the previous stuff didn't actually have any mention of templates haha

**Review by lindsayad — APPROVED** [review]

**Comment by aeslaughter** [comment]
> I can get on board with that. Although it seems like the previous stuff didn't actually have any mention of templates haha

It did, but I am guessing you updated it when you removed it.

**Comment by lindsayad** [comment]
Oh good job me!

**Pull request merged**